src/test/integration/elastic.e2e-spec.ts

```diff
@@ -0,0 +1,110 @@
+import { ElasticQuery } from 'src/common/elastic/entities/elastic.query';
+import { Test } from '@nestjs/testing';
+import { ElasticService } from 'src/common/elastic/elastic.service';
+import { PublicAppModule } from 'src/public.app.module';
+import { Constants } from 'src/utils/constants';
+import Initializer from './e2e-init';
+import { QueryConditionOptions } from 'src/common/elastic/entities/query.condition.options';
+
+describe('Elastic Service', () => {
+  let elasticService: ElasticService;
+
+  beforeAll(async () => {
+    await Initializer.initialize();
+
+    const moduleRef = await Test.createTestingModule({
+      imports: [PublicAppModule],
+    }).compile();
+
+    elasticService = moduleRef.get<ElasticService>(ElasticService);
+
+  }, Constants.oneHour() * 1000);
```

Comment:
This part should be deleted since for initalization we have a specific warm job

Reply:
Fixed